website/source/guides/web-ui/index.html.md

```diff
@@ -8,12 +8,14 @@ description: |-
 
 # Using Nomad with the Web UI
 
-The Nomad Web UI offers an easy to use web experience for inspecting a Nomad cluster. It's built
-into the Nomad binary and is served alongside the API. With zero additional configuration, you can
-use the Web UI instead of the CLI to inspect cluster state and submit and manage jobs.
+These guides have been migrated to [HashiCorp's Learn website].
 
-1. [Accessing the Web UI](/guides/web-ui/access.html)
-1. [Submitting a Job](/guides/web-ui/submitting-a-job.html)
-1. [Operating a Job](/guides/web-ui/operating-a-job.html)
-1. [Inspecting the Cluster](/guides/web-ui/inspecting-the-cluster.html)
-1. [Securing the Web UI with ACLs](/guides/web-ui/securing.html)
+You can follow these links to find the specific guides on Learn:
+
+- [Access the Web UI](https://learn.hashicorp.com/nomad/web-ui/access)
+- [Inspect Cluster Workload](https://learn.hashicorp.com/nomad/web-ui/ui-workload) (formerly "Operating a Job")
+- [Inspect the Cluster](https://learn.hashicorp.com/nomad/web-ui/inspecting-the-cluster)
+- [Submit a Job from the Web UI](https://learn.hashicorp.com/nomad/web-ui/submit-job)
+- [Nomad UI Considerations](https://learn.hashicorp.com/nomad/web-ui/considerations)
+
+[HashiCorp's Learn website]: https://learn.hashicorp.com/nomad?track=governance-and-policy#governance-and-policy
```

Comment:
This query param looks off.

Reply:
It's correct.  It just looks odd.  The track param sets the "active" track when the page renders.  The # jumps to the anchor itself.

> URI = scheme:[//authority]path[?query][#fragment]

Reply:
But why is it deep linking to the governance and policy track from the UI page?

Reply:
omg.  🤦‍♂   was doing so much work between the 2 tracks I missed what you meant.  fix incoming